src/Tests/FlatSharpCompilerTests/InvalidAttributeTests.cs

```diff
@@ -139,6 +139,6 @@ namespace ns;
         ";
 
         var ex = Assert.Throws<InvalidFbsFileException>(() => FlatSharpCompiler.CompileAndLoadAssembly(schema, new()));
-        Assert.Contains("Unable to parse 'fs_vector' value 'banana'. Valid values are: IList, IReadOnlyList, Array, Memory, ReadOnlyMemory, IIndexedVector.", ex.Message);
+        Assert.Contains("Unable to parse 'fs_vector' value 'banana'. Valid values are: IList, IReadOnlyList, Array, Memory, ReadOnlyMemory, IIndexedVector, NativeArray.", ex.Message);
```

Comment:
Reading this, I think I'd prefer the naming to be `Unity_NativeArray` or `UnityNativeArray`. I'd like to indicate that it's definitely a Unity-only thing.

Reply:
Renamed to `UnityNativeArray`